### PR TITLE
LEAN-2407

### PR DESCRIPTION
**Pull request by asouqi**

### CODE REVIEW

**Review by mnatsheh — APPROVED** [review]
Please change the version to 1.0.0.

**Comment by asouqi** [comment]
@mnatsheh could you please check `build.sh` and `extract-databases.js`, this changes to fix the build

File: scripts/extract-databases.js [hunk 1]
```diff
@@ -6,10 +6,6 @@ const path = require('path')
 
 fs.ensureDirSync('dist/shared')
 
-const customBundles = fs.readJSONSync('dist/shared/custom-bundles.json')
-const customTemplates = fs.readJSONSync('dist/shared/custom-templates.json')
-const publishedTemplates = fs.readJSONSync('dist/shared/published-templates.json')
-
 // NOTE: need to extract bundles before templates
 const files = ['bundles', 'templates-v2', 'symbols-v2', 'funders']
```
Comment (mnatsheh) [inline scripts/extract-databases.js]: Do we need symbols and funders here? we should also remove the db files.

**Review by mnatsheh — CHANGES_REQUESTED** [review]

File: scripts/extract-databases.js (hunk 1 above)
Comment (asouqi) [reply scripts/extract-databases.js]: I checked we don't use them at all, removed

**Review by mnatsheh — APPROVED** [review]

**Pull request merged**